packages/server/lib/controllers/sync.controller.ts

```diff
@@ -434,7 +435,7 @@ class SyncController {
         try {
             const { account, environment } = res.locals;
 
-            const { schedule_id, command, nango_connection_id, sync_id, sync_name, sync_variant, provider, delete_records } = req.body;
```

Comment:
can we stop passing those params in https://github.com/NangoHQ/nango/blob/7caf79bc921b195a3eddd1b66c7ac79b305bf4aa/packages/webapp/src/hooks/useSyncs.tsx#L20

Reply:
good point, I'll change that when migrating the endpoint